unicodetools/data/security/dev/data/confusablesSummaryIdentifier.txt

```diff
@@ -324,7 +324,7 @@
 
 #	bl	Ьl	Ь1	ЬІ	Ы
 	(‎ bl ‎)	0062 006C	 LATIN SMALL LETTER B, LATIN SMALL LETTER L
-←	(‎ Ьl ‎)	042C 006C	 CYRILLIC CAPITAL LETTER SOFT SIGN, LATIN SMALL LETTER L	# →ЬІ→→Ь1→
+←	(‎ Ьl ‎)	042C 006C	 CYRILLIC CAPITAL LETTER SOFT SIGN, LATIN SMALL LETTER L	# →Ь1→
```

Comment:
I was puzzled about this change in the comment. Neither of these characters were involved in the current set of changes. Possibly another upstream code change? I can investigate if it seems important to identify the cause.
cc: @markusicu @eggrobin

Reply:
I can’t say I understand these comments. You are adding things into the equivalence classes of both `b` and `l`, so maybe this has some effect on how this gets turned into `bl`?

I hope it is not the result of a code change, I haven’t done anything recently that should have an effect.